Test/Resolve-WhiskeyDotnetSdkVersion.Tests.ps1

```diff
@@ -210,3 +245,176 @@ Describe 'Resolve-WhiskeyDotNetSdkVersion.when Microsoft moves the index again'
         ThenReturnedNothing
     }
 }
+
+Describe 'Resolve-WhiskeyDotNetSdkVersion.when RollForward is diabled and patch is found' {
+    It 'should use specified version' {
+        Init
+        GivenReleases '3.0'
+        GivenSDKVersions '3.0.000', '3.0.100' -ForRelease '3.0'
+        WhenResolvingSdkVersion -Version '3.0.000' -RollForward Disable
+        ThenResolvedVersion '3.0.000'
+    }
+}
+
+Describe 'Resolve-WhiskeyDotNetSdkVersion.when RollForward is disabled and patch is not found' {
+    It 'should fail' {
+        Init
+        GivenReleases '3.0'
+        GivenSDKVersions '3.0.100' -ForRelease '3.0'
+        WhenResolvingSdkVersion -Version '3.0.000' -RollForward Disable
+        ThenError -Message 'not be found'
+    }
+}
+
+Describe 'Resolve-WhiskeyDotNetSdkVersion.when RollForward is patch and patch is found' {

```

Comment:
Create a loop to iterate over the same set of tests for the `patch`, `feature`, `minor`, `major` strategies since they are supposed to all behave the same.

```powershell
foreach ($strategy in @('patch', 'feature', 'minor', 'major'))
{
    Describe "Resolve-WhiskeyDotNetSdkVersion.when RollForward is $($strategy) and exact version does exist" {
        It 'should resolve latest patch' {
            Init
            GivenReleases '3.0'
            GivenSDKVersions '3.0.000', '3.0.001', '3.0.100' -ForRelease '3.0'
            WhenResolvingSdkVersion -Version '3.0.000' -RollForward $strategy
            ThenResolvedVersion '3.0.001'
        }
    }

    Describe "Resolve-WhiskeyDotNetSdkVersion.when RollForward is $($strategy) and exact version does not exist" {
        It 'should resolve latest patch' {
            Init
            GivenReleases '3.0'
            GivenSDKVersions '3.0.001', '3.0.002' -ForRelease '3.0'
            WhenResolvingSdkVersion '3.0.000' -RollForward $strategy
            ThenResolvedVersion '3.0.002'
        }
    }
}
```